capabilities/code-maintainability.md

```diff
@@ -5,28 +5,47 @@ In the lifespan of a typical line of code, it will likely be written once, edite
 As such, it makes sense to write code in a way that reduces the time required to comprehend it.
 Ideally, the software is built such that the reader can avoid needing to know how the whole system functions in order to make a local change.
 Furthermore, bugs are easier to spot when the surrounding code is easy to comprehend.
-The result of building code that's easy to maintain is more efficient delivery and better stability.
+The result of building code that's easy to maintain is more efficient delivery and better stability. But writing maintainable code requires an advanced set of skills — a set of skills that is developed by following the practices described in this section. 
 
-When difficult to maintain code is written, it can have a snowball effect.
-It requires an advanced set of skills to write maintainable code in a sea of hard to maintain code.
-Often, developers follow the path of least resistance by adding to the mess to get their new functionality added.
-There may be good reasons to follow such a path, however doing so further degrades the maintainability of the system.
-There is a delay between difficult to maintain code being introduced and the team experiencing a performance degradation.
-This delay makes it harder for non-technical team members to understand the consequences of implementing too many short-term solutions.
+<ed> Edit OK? I want to assure the reader that this "advanced set of skills" is indeed attainable and how. </ed>
+
+Too often, we are working with code that is difficult to maintain. 
+
+There are many reasons developers write code that is ultimately hard to maintain. 
+One key reason is that developers generally focus on learning skills that help them execute the job in front of them, like learning a language or framework. They tend to under-invest in skills that have a long-term payoff, like writing cleanly abstracted, easy to maintain, and extensible code. 
+
+<ed> Above text is taken from your reply to my query in the Gdoc. I like how you articulated this, so I added it to the intro. OK? </ed>
```

Comment:
Again, we'll want to remove the <ed> text from this file. But to answer the question, yes, this is ok.

Reply:
Excellent. Will remove the comment from the file.